src/components/Header.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import styled from 'styled-components';
+import logo from '../redditlogoOptimized.svg';
+
+const StyledHeader = styled.header`
+  padding-left: 80px;
+  padding-right: 80px;
+  padding-top: 32px;
+`;
+const StyledNav = styled.nav`
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+`;
+const StyledImg = styled.img`
```

Comment:
I wouldn't prefix the styled components with `Styled`. You can simply Image or better a bit more specific like `Logo` (not sure if this is the logo 😁)

Reply:
I agree, I was a bit leary about the naming too lol. Thanks, I'm gonna make those changes 👍  🙏